lib/common/strings.c

```diff
@@ -616,7 +616,7 @@ pcmk__parse_ll_range(const char *srcstring, long long *start, long long *end)
     *end = -1;
 
     crm_trace("Attempting to decode: [%s]", srcstring);
-    if (srcstring == NULL || strcmp(srcstring, "") == 0 || strcmp(srcstring, "-") == 0) {
+    if (pcmk__str_eq(srcstring, "", pcmk__str_null_matches) || strcmp(srcstring, "-") == 0) {
```

Comment:
This is an interesting case ... it combines the pcmk__str_eq() and pcmk__str_any_of() situations

Reply:
This might be better written as:
```
if (srcstring == NULL || pcmk__str_any_of(srcstring, "", "-") {
```